doc/tutorials/report.rst

```diff
@@ -56,6 +56,12 @@ To generate the report in parallel::
     mne report --path MNE-sample-data/ --info MNE-sample-data/MEG/sample/sample_audvis-ave.fif \ 
         --subject sample --subjects-dir MNE-sample-data/subjects --verbose --jobs 6
 
+The report rendered on sample-data is shown below:
+
+
+    .. image:: mne-report.png
```

Comment:
Ouch, 430 kB file added to the repo. Any way around this? Maybe a GIF, TIFF, JPEG, or PNG with compression would be smaller?